libcodechecker/server/client_db_access_handler.py

```diff
@@ -1731,56 +1731,41 @@ def massStoreRun(self, name, tag, version, b64zip, force):
         # Storing file contents from plist.
         file_path_to_id = {}
 
-        _, _, report_files = next(os.walk(report_dir), ([], [], []))
-        for f in report_files:
-            if not f.endswith('.plist'):
-                continue
-
-            LOG.debug("Parsing input file '" + f + "'")
-
-            try:
-                files, _ = plist_parser.parse_plist(
-                    os.path.join(report_dir, f))
-            except Exception as ex:
-                LOG.error('Parsing the plist failed: ' + str(ex))
+        for file_name, file_hash in filename2hash.iteritems():
```

Comment:
Is this python3 compatible? We try to not to introduce incompatibilities because in the future we want to support python 3 as well.